src/components/accounts/SetupSeedPhrase.js

```diff
@@ -104,19 +108,29 @@ class SetupSeedPhrase extends Component {
             location
         } = this.props
         const { recoveryKeyPair } = this.state
-
         const isNew = await checkIsNew(accountId)
 
         if (!isNew) {
-            await handleAddAccessKeySeedPhrase(accountId, recoveryKeyPair)
+            try {
+                await handleAddAccessKeySeedPhrase(accountId, recoveryKeyPair)
+                Mixpanel.track("SR-SP Verify seed phrase finish")
+            } catch(e) {
```

Comment:
The seed phrase has been `verified` at this point so here it's `setup`. Should also note that it's for `existing account`